indico/modules/events/timetable/models/entries.py

```diff
@@ -231,6 +231,15 @@ def end_dt(self):
     def end_dt(cls):
         return cls.start_dt + cls.duration
 
+    @property
+    def siblings(self):
+        parent = self.parent or self.event_new
+        day = self.start_dt.date()
```

Comment:
I think I already mentioned this on IRC, but don't you need to convert to event TZ here?